recipes/protobuf/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+    name: protobuf
+    version: "3.0.0a3"
```

Comment:
3.0.0a3 is not the most recent release of protobuf, looks like 3.0.0b2 is the most up to date, https://pypi.python.org/pypi/protobuf/3.0.0b2